gdsfactory/routing/manhattan.py

```diff
@@ -607,7 +608,7 @@ def round_corners(
     straight_fall_back_no_taper: ComponentSpec | None = None,
```

Comment:
I think the ComponentSpec for straight needs to be CellSpec instead now but otherwise appears to work